src/main/java/racingcar/domain/CarLocation.java

```diff
@@ -0,0 +1,44 @@
+package racingcar.domain;
+
+import java.util.Objects;
+
+public class CarLocation {
+    private int location;
+
+    public CarLocation() {
+        this.location = 0;
```

Comment:
주 생성자를 호출하는 방식으로 변경해보면 어떨까요?
```suggestion
        this(0);
```